src/executor/stack/mod.rs

```diff
@@ -79,10 +80,17 @@ impl<'config> StackSubstateMetadata<'config> {
 	}
 }
 
+pub struct PrecompileOutput {
+	pub exit_status: ExitSucceed,
+	pub output: Vec<u8>,
+	pub cost: u64,
```

Comment:
Place the fixed-size `cost` field prior to the dynamic, variable-length `output` field.

Reply:
Done